core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -8153,7 +8154,7 @@ public TesterImpl(SqlTestFactory testFactory) {
       } catch (RuntimeException e) {
         throw e;
```

Comment:
I think this catch "RuntimeException" is not needed; rethrow could be able to handle that.

I was wondering...I usually prefer to throw Exception from test methods/classes - but that change in this case seems to imply around 100 "throws Exception" additions :) so it might be better to leave it this way

Reply:
That's true, it should be removed. I just did a global search and replace, and I did not verify the individual cases.

It seems to be the only entry in test code for `catch \(RuntimeException \w+\) \{\s*throw` regexp.

Reply:
>I usually prefer to throw Exception from test methods/classes

Just to clarify: you can't `throw Exception` from a lambda. This change is mostly to unlock that.